shared/common-adapters/input.desktop.js

```diff
@@ -114,7 +114,6 @@ export default class Input extends Component {
           rows={this.props.rows}
           rowsMax={this.props.rowsMax}
           style={{...textStyle, ...globalStyles.flexBoxColumn, ...this.props.textStyle}}
-          textAlign='center'
```

Comment:
This is the propType error that I was talking about. It also appears redundant/less-accurate to the textAlign that is being set in the style.